addons/mail/models/mail_channel.py

```diff
@@ -985,17 +985,20 @@ def _execute_command_help(self, **kwargs):
             all_channel_partners = self.env['mail.channel.partner'].with_context(active_test=False)
             channel_partners = all_channel_partners.search([('partner_id', '!=', partner.id), ('channel_id', '=', self.id)])
             msg = _("You are in a private conversation with <b>@%s</b>.") % (channel_partners[0].partner_id.name if channel_partners else _('Anonymous'))
-        msg += _("""<br><br>
-            Type <b>@username</b> to mention someone, and grab his attention.<br>
-            Type <b>#channel</b> to mention a channel.<br>
-            Type <b>/command</b> to execute a command.<br>
-            Type <b>:shortcut</b> to insert a canned response in your message.<br>""")
+        msg += self._execute_command_help_message()
 
         self._send_transient_message(partner, msg)
 
     def _define_command_leave(self):
         return {'help': _("Leave this channel")}
 
+    def _execute_command_help_message(self):
+        msg = _("""<br><br>
+            Type <b>@username</b> to mention someone, and grab his attention.<br>
+            Type <b>#channel</b> to mention a channel.<br>
+            Type <b>/command</b> to execute a command.<br>""")
+        return msg
+
```

Comment:
Considering name of function, it should contain the whole message, including the 1st sentence.

Reply:
I am little bit confused that do i need to change the name of the function or need to includ the code which gives 1st sentence into this function?

Reply:
Hello...Should we move the stuff which gives 1st sentence in newly created function from existing "_execute_command_help"? (https://github.com/odoo/odoo/blob/master/addons/mail/models/mail_channel.py#L981)

Reply:
Hm... Or just add "extra" as suffix of the name function, and it should be good.
I just didn't like that this function is named as if it made the whole message, but in practice it's just the listing of special characters.

Reply:
Changes done..Thanks..